src/scripts/clipperUI/components/annotationInput.tsx

```diff
@@ -47,7 +47,7 @@ class AnnotationInputClass extends ComponentBase<{}, ClipperStateProp> {
 					</span>
 					<br/>
 				</pre>
-				<textarea
+				<input
```

Comment:
Is there no way we can change the placeholder color for textarea from css? because changing tags might have other issues as well such as input is a focusable element etc.

Reply:
There's a dead code in css for annotation placeholder and initially I was trying to add the new items there, so didn't check specifically for textarea once this worked. Have reverted back to the tag and updated css. 
Also, textarea and input are both focusable elements with similar properties, so I don't think there should be much difference.